api/sessions/connection.gen.go

```diff
@@ -0,0 +1,14 @@
+// Code generated by "make api"; DO NOT EDIT.
+package sessions
+
+type Connection struct {
+	PublicId           string `json:"public_id,omitempty"`
```

Comment:
I have a concern regarding the resource organization itself... I added a comment on PR 1714 but to fix the broken build I'll go ahead and approve this.